### PR TITLE
Add Subscription Summary Metrics endpoint

### PR DESCRIPTION
**Pull request by pmeth**

### CODE REVIEW

File: spec/definitions/SubscriptionSummaryMetrics.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  invoiced:
+    description: "Total amount invoiced. Formatted according to the currency, example: $23.02"
+    readOnly: true
+    type: string
+  collected:
+    description: "Total amount paid. Formatted according to the currency, example: $15.85"
+    readOnly: true
+    type: string
+  numberOfInvoices:
+    description: Number of active invoices
```
Comment (adamaltman) [inline spec/definitions/SubscriptionSummaryMetrics.yaml]: active?

Comment (pmeth) [reply spec/definitions/SubscriptionSummaryMetrics.yaml]: I am filtering out `draft`, `voided` and `abandoned` statuses so I wanted a way to describe that. I suppose it is misleading language? What do you think?

Comment (adamaltman) [reply spec/definitions/SubscriptionSummaryMetrics.yaml]: Maybe:
> Number of issued, past due, or paid invoices

File: spec/definitions/SubscriptionSummaryMetrics.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    type: string
+  invoiced:
+    description: "Total amount invoiced. Formatted according to the currency, example: $23.02"
+    readOnly: true
+    type: string
+  collected:
+    description: "Total amount paid. Formatted according to the currency, example: $15.85"
+    readOnly: true
+    type: string
+  numberOfInvoices:
```
Comment (2e3s) [inline spec/definitions/SubscriptionSummaryMetrics.yaml]: Maybe `invoicesCount`? It seems more consistent with what we have.

File: spec/definitions/SubscriptionSummaryMetrics.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    type: string
+  collectedAmount:
+    description: "Total amount paid. Formatted according to the currency, example: $15.85"
+    readOnly: true
+    type: string
+  invoiceCount:
+    description: Number of issued, past due, or paid invoices
+    readOnly: true
+    type: number
+    format: int
```
Comment (2e3s) [inline spec/definitions/SubscriptionSummaryMetrics.yaml]: There is no such a format: https://swagger.io/docs/specification/data-models/data-types/

File: spec/definitions/SubscriptionSummaryMetrics.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+type: object
+properties:
+  currency:
+    description: Currency (three letter ISO 4217 alpha code) (eg USD, EUR)
+    readOnly: true
+    type: string
+  invoicedAmount:
+    description: "Total amount invoiced. Formatted according to the currency, example: $23.02"
```
Comment (2e3s) [inline spec/definitions/SubscriptionSummaryMetrics.yaml]: > Formatted according to the currency, example: $23.02"

I think it should be just a float number. Formatting is a client concern.

File: spec/definitions/SubscriptionSummaryMetrics.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  collectedAmount:
+    description: Sum of all paid invoices
+    readOnly: true
+    type: number
+    format: double
+  invoiceCount:
+    description: Number of issued, past due, or paid invoices
+    readOnly: true
+    type: number
+    format: integer
```
Comment (2e3s) [inline spec/definitions/SubscriptionSummaryMetrics.yaml]: No, I mean `type: integer`. `format: integer` is not listed in the specs

**Pull request merged**